diagnostic.md

```diff
@@ -150,7 +183,7 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divFour divTwo divThree divFive divOne
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short:
```
2 3      1
4 5